src/openforms/formio/rendering/default.py

```diff
@@ -345,3 +346,91 @@ def is_visible(self) -> bool:
         registration data...
         """
         return False
+
+
+class PartnerValue(TypedDict):
+    bsn: str
+    initials: NotRequired[str]
+    affixes: NotRequired[str]
+    first_names: NotRequired[str]
+    last_name: NotRequired[str]
+    date_of_birth: NotRequired[str]
+    date_of_birth_precision: Literal["date", "year_month", "year"] | None
+
+
+@register("partners")
+class PartnersNode(ComponentNode):
+    @property
+    def value(self) -> Any:
```

Comment:
this can probably be the proper return type?


```suggestion
    def value(self) -> Sequence[PartnerValue]:
```